GOVERNANCE.md

```diff
@@ -0,0 +1,62 @@
+# Scope Governance
+
+This document defines project governance for the project. This is (and probably will always be) a Work in Progress.
+
+## Goals and Principles
+
+Scope's community goals and principles:
+
+1. Transition from a primarily Weaveworks project to a true community-driven project with autonomous governance.
```

Comment:
What does this mean exactly?

A community's mission should more about furthering the use and broadening the scope (hah!) of scope and making its community be friendly and tick along nicely, ... or something along the lines.

Are there concrete issues the move to a more "community-driven" model fixes right now?

Reply:
I don't think there are specific goals other than nurture the community and see where it wants to go by itself (hopefully increasing use as a side effect). We noticed interest from the various actors, an in-person meeting happened at Kubecon (organized by @monadic ) which confirmed that interest and we want to see where the community can lead the project.

Reply:
Excellent. Those (more general) goals sound like a good "mission statement" or charter for the project.

Reply:
OK, I will incorporate them.